index.html

```diff
@@ -1,39 +1,348 @@
+          </div>
+        </div>
 
-        
+        <div class="project">
+          <div class="project-header">
+            <h1>Professional Art Printing Data</h1>
+          </div>
+          <div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <div>
+            <ul class="projectLanguages">
+              <li><img src="./images/Tag (4).png" alt="" /></li>
+              <li><img src="./images/Tag (5).png" alt="" /></li>
+              <li><img src="./images/Tag (6).png" alt="" /></li>
+            </ul>
+          </div>
+          <div>
+            <button>See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
 
+    <section class="about-me">
+      <div class="about">
+        <div class="about-me-details">
+          <h1 class="headline">About me</h1>
+          <p class="introduction">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <button>Get My Resume</button>
+        </div>
+      </div>
+      <div class="about-line">
+        <hr />
+      </div>
+      <div class="aboutList">
+        <ul>
+          <img src="./images/Rectangle 55.png" alt="" />
+          languages
+          <li>Javascript</li>
+          <li>ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+        <ul>
+          <img src="./images/Rectangle 56.png" alt="" />
+          Frameworks
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>Rspec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+        <ul>
+          <img src="./images/Rectangle 57.png" alt="" />
+          Skills
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
+    <section>
+      <div class="contact-us">
+        <div class="contact-text">
+          <h1>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h1>
+        </div>
+        <div class="contactUs-form">
+          <form
+            class="contact-form"
+            action="https://formspree.io/f/mzbwnvpb"
+            method="POST"
+          >
+            <input
+              class="input-field"
+              type="text"
+              maxlength="30"
+              required
+              placeholder="Full names"
+            />
+            <input
+              class="input-field"
+              type="email"
+              required
+              placeholder="email address"
```

Comment:
- [ ] Kindly make sure the contact section stick to the design template .

Your Design
![Screenshot from 2022-12-10 18-05-51](https://user-images.githubusercontent.com/80458199/206865840-f29d108c-4270-4ee1-ae21-81e81ebd0c8a.png)


Figma Design

![Screenshot from 2022-12-10 18-14-55](https://user-images.githubusercontent.com/80458199/206865831-eaf9fc8a-f9b6-4d6a-89f3-86f074810d0f.png)

Reply:
Fixed.